.github/workflows/update_rubocop.yml

```diff
@@ -0,0 +1,38 @@
+
+name: Update Rubocop
+
+on:
+  # Can be triggered manually from the GH interface or via a dispatches POST
+  # https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
+  workflow_dispatch:
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+  # Every sunday at 10 am
```

Comment:
How does "0 10 * * 0" specify every sunday at 10 🤔

Reply:
The link suggests it gets built off the default branch, which is develop, is that okay or should it be of master?

Reply:
Welcome to the wonders of cron syntax. I need to use a helper tool online every time I need to write one.
Think I used this on here: https://crontab.guru
Yep, develop is the intent. Means we see the advantages earlier as developers, but the risks hit the users later.